Code/Framework/AzNetworking/AzNetworking/AutoGen/AutoPackets_Header.jinja

```diff
@@ -21,7 +21,7 @@ AZStd::fixed_vector<{{ element.attrib['Type'] }}, {{ element.attrib['Count'] }}>
         static constexpr AzNetworking::PacketType Type = aznumeric_cast<AzNetworking::PacketType>({{ type }});
 
         {{ name }}() = default;
-{%  if (packetNode.getchildren()) | len > 0 %}
```

Comment:
```getChildren``` is deprecated for xml.etree.cElement in python 3.10 (see https://github.com/python/cpython/issues/80724 )